### PR TITLE
Documentation update for #272 feature.

### PR DESCRIPTION
**Pull request by ManrajGrover**
Hi @rxaviers sir @jzaefferer sir,

Here is the documentation update for new #272 feature. I have added few examples explaining the use of this feature. Let me know further changes. Thank you.

### CODE REVIEW

File: doc/api/date/date-formatter.md [hunk 1]
```diff
@@ -168,6 +168,17 @@ Use skeletons for more flexibility (see its description [above](#parameters)).
...
+
+| `skeleton` | `Globalize( "en" ).dateFormatter( skeleton )( new Date( 2010, 10, 1, 17, 55 ) )` |
+| --- | --- |
+| `"GyMMMEdhms"` | `"Mon, Nov 1, 2010 AD, 5:55:00 PM"` |
+| `"Ehms"` | `"Mon 5:55:00 PM"` |
+| `"yQQQHm"` | `"Q4 2010, 17:55"` |
+| `"MMMEdhm"` | `"Mon, Nov 1, 5:55 PM"` |
+| `"yMMMdhm"` | `"Nov 1, 2010, 5:55 PM"` |
+
+
```
Comment (rxaviers) [inline doc/api/date/date-formatter.md]: The fact there are date and field parts in the same skeleton should be transparent for the end-user. Let's have your examples merged in the above table. Also, feel free to update the previous paragraph to include "(in Canonical Order)".

**Comment by rxaviers** [comment]
Thanks. I left a comment.

**Comment by ManrajGrover** [comment]
Hi @rxaviers sir, :)

I have merged and updated the description. Please review.

**Comment by rxaviers** [comment]
Thanks @ManrajGrover

**Pull request closed without merging**